hack/test-cmd.sh

```diff
@@ -586,7 +586,7 @@ oc delete bc/ruby-sample-build-validtag
 oc delete bc/ruby-sample-build-invalidtag
 
 # Test admin manage-node operations
-[ "$(openshift admin manage-node --help 2>&1 | grep 'Manage node operations')" ]
+[ "$(atomic-enterprise admin manage-node --help 2>&1 | grep 'Manage node operations')" ]
```

Comment:
I took a look and both openshift admin/atomic-enterprise admin and oadm produce the same results.